build.sh

```diff
@@ -30,7 +30,8 @@ on_exit() {
 }
 
 trap on_exit EXIT
-
+# as executable bit will be inherited to the docker image we explicitly set executable bit on scripts
+chmod start-entrypoint.d/* before-migrate-entrypoint.d/*
```

Comment:
didn't see it before, the mode is missing :)

```suggestion
chmod +x start-entrypoint.d/* before-migrate-entrypoint.d/*
```

Reply:
Sry, fixed